artman/tasks/gapic_tasks.py

```diff
@@ -112,6 +138,35 @@ def validate(self):
         return [gapic_requirements.GapicRequirements]
 
 
+class DiscoGapicCodeGenTask(task_base.TaskBase):
+    """Generates GAPIC wrappers from a Discovery document"""
+    default_provides = 'gapic_code_dir'
+
+    def execute(self, language, toolkit_path, discovery_doc,
+        gapic_api_yaml, discogapic_language_yaml, package_metadata_yaml,
+        gapic_code_dir, api_name, api_version, organization_name):
+        existing = glob.glob('%s/*' % gapic_code_dir)
+        if existing:
+            self.exec_command(['rm', '-r'] + existing)
+        gapic_yaml = gapic_api_yaml + discogapic_language_yaml
+        gapic_args = ['--gapic_yaml=' + os.path.abspath(yaml)
+                      for yaml in gapic_yaml]
+        args = [
+            # TODO(andrealin): Get right absolute path for discovery_doc
```

Comment:
This todo might be done? (Not obvious to me.)

Reply:
I'm not sure why this doesn't work either :(
I just know that in googleapis/.../artman_compute.yaml, I have to set
`discovery_doc: google/compute/v1/compute.v1.json`
even though I would prefer, in line with convention, to set
`discovery_doc: v1/compute.v1.json`